OpenDreamRuntime/Procs/Native/DreamProcNative.cs

```diff
@@ -8,6 +11,50 @@ static class DreamProcNative {
             DreamProcNativeRoot.DreamManager = IoCManager.Resolve<IDreamManager>();
 
             DreamObjectDefinition root = objectTree.GetObjectDefinition(DreamPath.Root);
+
+            Assembly asm = Assembly.GetExecutingAssembly();
+            
+            var q = from t in Assembly.GetExecutingAssembly().GetTypes()
+                    where t.Name == "DreamProcNativeRoot" && t.IsClass && t.Namespace == "OpenDreamRuntime.Procs.Native"
+                    select t;
+
+            foreach (var t in q)
+            {
+                foreach (var memberI in t.FindMembers(MemberTypes.Method, BindingFlags.Public | BindingFlags.Static, null, null)) {
```

Comment:
You can use `t.GetMethods(BindingFlags.Public | BindingFlags.Static)` here to avoid having to get `methodI` in a second step later on.